data_checker/README.md

```diff
@@ -0,0 +1,40 @@
+# Check of correct import of resource policies
+### We check ONLY if ANONYMOUS group has same READ access to ITEMS in Dspace5 and Dspace7 after success data import! We don't check other resource policies.
+
+## How to use it:
+1. Update `const.py`
+   - `user = "<ADMIN_NAME>"`

```

Comment:
No need for this, we are not using authentication. Do not instruct user to set variables that are not used. It is confusing. (same goes for password of course)